libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -739,19 +761,25 @@ ParseFlags parseFlag(CompileCommand cmd, const CompileCommandFilter flag_filter,
                 st = State.keep;
             } else if (st == State.pathArgumentToAbsolute) {
                 st = State.keep;
-                auto p = buildNormalizedPath(workdir, arg).absolutePath;
-                rval.put(p);
-                includes.put(p);
+                putNormalizedAbsolute(arg);
             } else if (st == State.priorityKeepNextArg) {
                 st = State.keep;
                 rval.put(arg);
+	    } else if (st == State.checkingForQuotation) {
+		path.put(" " ~ arg);
+		if (isEndingWithQuotation(arg)) {
+		  st = State.keep;
+		  putNormalizedAbsolute(path.data.strip('"'));
+		}
             } else if (excludeStartWith(arg, flag_filter)) {
                 st = State.skipIfNotFlag;
-            } else if (isCombinedIncludeFlag(arg)) {
+	    } else if (isSpacedInclude(arg)) {
```

Comment:
Indention is a bit wonky here

Reply:
I know, I wrote it earlier that my formatchecker was acting up.